types/artifacts.d.ts

```diff
@@ -553,12 +553,21 @@ declare module Artifacts {
   }
 
   interface InspectorIssues {
```

Comment:
Of course. When I saw that this change to the gatherer breaks a bunch of other stuff, I thought it might be better to break this into it's own PR. 

Would it be better to update this PR with all of the work?

Reply:
Keeping it separate is the preference. I just wanted to write down the new items